docs/TEXTER_ACTIVITY_QUERIES.md

```diff
@@ -99,10 +95,53 @@ GROUP BY
 "user".id
 ORDER BY
   message_sent_count DESC;
+`
+
+* counts of texters, sent, responses, total texts, opt outs and estimated cost for all current campaigns
+`SELECT
+    c.id AS campaign_id,
+    c.title AS campaign,
+    texters::int,
+    texts_sent::int,
+    replies::int,
+    total_texts::int,
+    opt_outs::int,
+    /* cost per texter = $1 / 30 days (est) */
+    ROUND(((texters / 30)  + ((total_texts * .015)*1.15)), 2) AS costid
```

Comment:
hey @joemcl - where did you get this `.015` cost from? can you make a note that this is factoring text messages to be 2 segments (so 2 x .0075)? thanks!

Reply:
yup will do @shakalee14